File: backend/model/lpd.go
package model

type Lpd struct {
	Id            uint   `json:"id" gorm:"primaryKey"`
	SuratId       uint   `json:"surat_id" gorm:"not null"`
	HasilKegiatan string `json:"hasil_kegiatan" gorm:"not null"`
	Dokumentasi   string `json:"dokumentasi" gorm:"not null"`
	PetugasId     uint   `json:"petugas_id" gorm:"not null"`

	Surat Surat `json:"surat" gorm:"foreignKey:SuratId;references:Id"`
}

func (Lpd) TableName() string {
	return "lpd"
}

File: backend/app/Models/Lpd.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Lpd extends Model
{
    use HasFactory;

    protected $table = 'lpd';

    public $timestamps = false;

    protected $fillable = [
        'surat_id',
        'hasil_kegiatan',
        'dokumentasi',
    ];

    /**
     * Relasi ke Surat (Many LPD belongs to One Surat)
     */
    public function surat()
    {
        return $this->belongsTo(Surat::class, 'surat_id');
    }

    /**
     * Relasi Many-to-Many ke PetugasLain
     * lpd_petugas_lain (pivot)
     */
    public function petugasLain()
    {
        return $this->belongsToMany(Petugas_Lain::class, 'lpd_petugas_lain', 'lpd_id', 'petugas_lain_id');
    }
}
